application/uitests/org.openjdk.jmc.console.uitest/src/test/java/org/openjdk/jmc/console/uitest/WelcomePageTest.java

```diff
@@ -45,9 +45,9 @@
  * and closed.
  */
 public class WelcomePageTest extends MCJemmyTestBase {
-	
+
 	/**
-	 * Open and close the Welcome page 
+	 * Open and close the Welcome page
```

Comment:
Can spotless be configured to either require that a sentence start with lowercase or ends with a period?

Reply:
I don't think so, but I'd like to keep it so that if it has been formatted from within Eclipse, it should pass. We can add more sophisticated stuff, including XML file formatting, later.